packages/core/src/runtime/resolveStyleRules.ts

```diff
@@ -54,6 +55,14 @@ export function resolveStyleRules(
 ): [CSSClassesMap, CSSRulesByBucket] {
   // eslint-disable-next-line guard-for-in
   for (const property in styles) {
+    // eslint-disable-next-line no-prototype-builtins
+    if (UNSUPPORTED_CSS_PROPERTIES.hasOwnProperty(property)) {
+      if (process.env.NODE_ENV !== 'test' && process.env.NODE_ENV !== 'production') {
```

Comment:
Can you please clarify why we should not warn in tests?

Reply:
it was polluting tests, but I can disable console just for that specific test in griffel

Reply:
Yes, please 👍

Reply:
babel-preset tests are kinda awkward to mock console, any ideas ?

Reply:
We had an offline call about it, implemented in [1d8a02d](https://github.com/microsoft/griffel/pull/47/commits/1d8a02d1d848b7dab1c24a9ec9f3105692fd8d4f)